django-cloudlaunch/cloudlaunch/backend_plugins/cloudman2/rancher2_aws_iam_trust_policy-cn.json

```diff
@@ -0,0 +1,12 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ec2.amazonaws.com"
```

Comment:
Shouldn't this address end in `.cn` rather than the default file below?

Reply:
Thanks for the catch! Fixed.